clang-tools-extra/docs/clang-tidy/checks/hooks-guard-in-for.rst

```diff
@@ -6,10 +6,49 @@ hooks-guard-in-for
 Consider the following for-loop in C:
 
 .. code-block:: c
-
+  
```

Comment:
These files are no longer used, so instead of updating, it's simpler to just empty them (not remove, b/c LLVM build might still expect to find them). Up-to-date docs should be maintained in https://github.com/XRPLF/xrpl-hooks-ide/tree/main/xrpl-hooks-docs/md ...